apps/dashboard/src/pages/Root.tsx

```diff
@@ -22,7 +49,14 @@ const Root: React.FC = () => {
   }, [location.pathname, navigate, profile]);
 
   return (
-    <main className="min-h-screen text-foreground flex overflow-y-hidden">
+    <main
+      className={`grid ${
+        !profile ? "" : "grid-cols-[5%,95%]"
+      } min-h-screen overflow-y-hidden text-foreground`}
```

Comment:
Please use the `classnames` library to add conditional class names. You can look into other files through out the codebase to get a sense on how to use it.